### ISSUE TITLE
Unable to expand TV Show in Media Info

### ISSUE THREAD

**Issue by gardinerbk** [Tautulli/Tautulli#486]
One show is not able to expand in the Media Info tab of TV Shows in Libraries. Its five seasons are detected when clicking on the show for more details. 

![expandingproblem](https://cloud.githubusercontent.com/assets/15112109/12576515/8320961a-c414-11e5-9566-c9de1b87231a.PNG)

**Comment by JonnyWong16** [comment]
Have you tried clicking the "Refresh media info" button?

**Comment by gardinerbk** [comment]
Yes I tried that.

**Comment by JonnyWong16** [comment]
Try deleting the cache for that library table by going to:

```
http://localhost:8181/delete_datatable_media_info_cache?section_id=<section_id>
```

Where `<section_id>` can be found in the library url.

**Comment by gardinerbk** [comment]
Did that just now and it didn't change anything. It's not bothering me by the way. I just thought that I should report it. Don't know if it is a general thing or just something that went wrong on my side.

**Comment by JonnyWong16** [comment]
One more thing to check. Go to 

```
http://<plex_ip>:32400/library/metadata/<rating_key>?X-Plex-Token=<your_plex_token>
```

You can find the `<rating_key>` in the URL of the TV show in PlexPy.

See if the TV show has an `addedAt` time in the XML.

**Comment by gardinerbk** [comment]
No added at.

`<MediaContainer size="1" allowSync="1" identifier="com.plexapp.plugins.library" librarySectionID="2" librarySectionTitle="TV Shows" librarySectionUUID="f7e97c6b-be24-444a-b89e-2b3aeac0f85f" mediaTagPrefix="/system/bundle/media/flags/" mediaTagVersion="1453456071">`

`<Directory ratingKey="5319" key="/library/metadata/5319/children" guid="com.plexapp.agents.thetvdb://247897?lang=en" librarySectionID="2" studio="Showtime" type="show" title="Homeland" contentRating="TV-MA" summary="It has been 2 years since the events in Pakistan. Carrie Mathison has left the CIA and is working as head of security for the philanthrophic Düring Foundation and residing in Berlin, Germany. Two hackers from Berlin accidentally gain access to the CIA network and get their hands on sensitive documents regarding Germany allowing the CIA to snoop on German citizens. " index="1" rating="8.3" viewCount="60" lastViewedAt="1450731448" year="2011" thumb="/library/metadata/5319/thumb/1450662203" art="/library/metadata/5319/art/1450662203" banner="/library/metadata/5319/banner/1450662203" theme="/library/metadata/5319/theme/1450662203" duration="2700000" originallyAvailableAt="2011-10-02" leafCount="60" viewedLeafCount="60" childCount="5" updatedAt="1450662203">`
`<Genre id="93" tag="Crime"/>`
`<Genre id="24" tag="Drama"/>`
`<Genre id="18263" tag="Suspense"/>`
`<Genre id="25" tag="Thriller"/>`
`<Role id="9539" tag="Rupert Friend" role="Peter Quinn"/>`
`<Role id="18264" tag="Mandy Patinkin" role="Saul Berenson"/>`
`<Role id="9279" tag="Claire Danes" role="Carrie Mathison"/>`
`<Role id="7899" tag="F. Murray Abraham" role="Dar Adal"/>`
`<Role id="17533" tag="Damian Lewis" role="Nicholas Brody"/>`
`<Role id="4622" tag="Miranda Otto" role="Allison Carr"/>`
`<Role id="21481" tag="Sarah Sokolovic" role="Laura Sutton"/>`
`<Role id="17486" tag="Sebastian Koch" role="Otto Düring"/>`
`<Role id="18265" tag="Numan Acar" role="Haissam Haqqani"/>`
`<Role id="7107" tag="Nimrat Kaur" role="Tasneem Qureshi"/>`
`<Role id="12836" tag="Suraj Sharma" role="Aayan Ibrahim"/>`
`<Role id="18266" tag="Laila Robins" role="Martha Boyd"/>`
`<Role id="7462" tag="Sarita Choudhury" role="Mira Berenson"/>`
`<Role id="18267" tag="Tracy Letts" role=" Senator Andrew Lockhart"/>`
`<Role id="14407" tag="Nazanin Boniadi" role="Fara Sherazi"/>`
`<Role id="18268" tag="Jackson Pace" role="Chris Brody"/>`
`<Role id="11901" tag="Diego Klattenhoff" role="Mike Faber"/>`
`<Role id="18269" tag="David Harewood" role="David Estes"/>`
`<Role id="10908" tag="Morena Baccarin" role="Jessica Brody"/>`
`<Role id="18270" tag="Morgan Saylor" role="Dana Brody"/>`
`<Field name="thumb" locked="1"/>`
`<Location path="---"/>`
`</Directory>`
`</MediaContainer>`

**Comment by JonnyWong16** [comment]
This a Plex problem then. Plex doesn't have the date the TV show was added to your library.

**Comment by gardinerbk** [comment]
Alright I'll see if I can fix that. Thank you!!

**Comment by gardinerbk** [comment]
I removed and added the TV Show and that fixed it. Lost my stats on the show though, but it works now.

**Comment by JonnyWong16** [comment]
Click on one of the episodes in the history table, then click on "Fix Metadata".

**Comment by gardinerbk** [comment]
Cool that fixed it!

**Comment by JonnyWong16** [comment]
Fixed in v1.3.5.
